hawk/src/main/java/com/orhanobut/hawk/Hawk.java

```diff
@@ -56,19 +61,7 @@ public static void init(Context context, String password, LogLevel logLevel) {
      * @return true if put is successful
      */
     public static <T> boolean put(String key, T value) {
-        if (key == null) {
```

Comment:
For the time being, let's keep the structure as it is. We can discuss the best practice for the validation with a new PR. Current PR should be focused on the chain feature only.